integration_tests/utils.py

```
@@ -970,31 +970,18 @@ def assert_register_erc20_denom(c, addr, tmp_path):
     fee = receipt["effectiveGasPrice"] * receipt["gasUsed"]
-    assert await balance_of(w3, weth_addr, owner) == 1000
-    receipt = await weth.fns.withdraw(1000).transact(w3, account)
-    fee += receipt["effectiveGasPrice"] * receipt["gasUsed"]
-    assert await balance_of(w3, weth_addr, owner) == 0
-    assert await balance_of(w3, ZERO_ADDRESS, owner) == before - fee
-
-    # withdraw should be nop
-    before = await balance_of(w3, ZERO_ADDRESS, owner)
-    receipt = await weth.fns.deposit().transact(w3, account, value=1000)
-    fee = receipt["effectiveGasPrice"] * receipt["gasUsed"]
     await balance_of(w3, weth_addr, owner) == 1000
```

Ruff (0.12.2)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)